assets/blocks/course-progress-block/course-progress-edit.js

```diff
@@ -76,17 +76,17 @@ export const CourseProgressEdit = ( props ) => {
 			borderRadius,
 		},
 	};
-
+	const progressBarLabel = _n( 'lesson', 'lessons', 2, 'sensei-lms' );
 	return (
 		<>
 			<ProgressBar
-				lessonsCount={ totalLessonsCount }
+				totalCount={ totalLessonsCount }
 				completedCount={ completedLessonsCount }
 				wrapperAttributes={ wrapperAttributes }
 				barWrapperAttributes={ barWrapperAttributes }
 				barAttributes={ barAttributes }
 				countersClassName="wp-block-sensei-lms-progress-heading"
-				lessonsCountClassName="wp-block-sensei-lms-progress-heading__lessons"
+				progressBarLabel={ progressBarLabel }
```

Comment:
I think we could call this prop `label` and drop the `progressBar` prefix since it's already part of the `ProgressBar` component.